src/components/ProjectList/ProjectItem.js

```diff
@@ -40,98 +58,73 @@ class ProjectItem extends Component {
   render() {
     const {
       item,
-      childrenProp,
       primaryTextProp,
       secondaryTextProp,
       valueProp,
       clickHandler,
+      updated,
+      owner,
       inset,
     } = this.props
 
     const primaryText = item[primaryTextProp]
     const secondaryText = secondaryTextProp ? item[secondaryTextProp] : ''
     const value = item[valueProp]
-    const children = item[childrenProp]
-
-    const listItemWithNestedStyle = {
-      fontSize: '13px',
-    }
+    const ownerName = item[owner].first_name + ' ' + item.owner.last_name // eslint-disable-line
+    const updatedTime = item[updated]
+    // TODO: something else
```

Comment:
For string concatenation, use template literals https://developer.mozilla.org/en/docs/Web/JavaScript/Reference/Template_literals